pkg/utils/history_buffer.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import "fmt"
+
+type HistoryBuffer[T any] struct {
+	maxSize int
+	Items   []T
```

Comment:
```suggestion
	items   []T
```
This ought to be private

Reply:
good call

fixed in: 8e73ffe7d5f43b0b51a293c0c3096f93d5966392